pkgs/by-name/cr/crystal-dock/package.nix

```
@@ -0,0 +1,42 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  kdePackages,
+  qt6,
+}:
+stdenv.mkDerivation rec {
+  pname = "crystal-dock";
+  version = "2.2";
+
+  src = fetchFromGitHub {
+    owner = "dangvd";
+    repo = "crystal-dock";
+    rev = "v${version}";
+    sha256 = "sha256-c5Kae2cZ/DoJ972VT4kQWNUr2cF6Noy3nPIChWok/BA=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    kdePackages.extra-cmake-modules
+    qt6.wrapQtAppsHook
+  ];
+
+  buildInputs = [
+    kdePackages.layer-shell-qt
+    qt6.qtbase
+    qt6.qtwayland
+  ];
+
+  cmakeDir = "../src";
+
+  meta = with lib; {
+    description = "Cool dock (desktop panel) for Linux desktop.";
+    mainProgram = "crystal-dock";
+    license = licenses.gpl3Only;
+    homepage = "https://github.com/dangvd/crystal-dock";
+    maintainers = with maintainers; [ rafameou ];
+    platforms = platforms.linux;
+  };
+}
```

Close the newly opened parenthesis before `finalAttrs` on the `mkDerivation`-line:
```suggestion
})
```